packages/hiro-graph-client/index.d.ts

```diff
@@ -425,11 +427,17 @@ interface DefineVariableOptions {
   [key: string]: any;
 }
 
+interface SuggestVariable {
+  name: string;
+  full?: boolean;
+  [key: string]: any;
+}
+
```

Comment:
What do you think, any sense to extend PlainObject here to support `[key: string]: any;`?

Reply:
done